src/components/web3Connection/WalletSelectorModal/WalletSelectorModal.tsx

```diff
@@ -55,58 +58,66 @@ const Web3Modal = ({
   const handleOnboarding = () => onboarding.current?.startOnboarding()
 
   useEffect(() => {
-    if (active) {
+    if (active) closeModal()
+  }, [active, closeModal])
+
+  useEffect(() => {
```

Comment:
This was a great idea too. Should we maybe show a similar modal when the user navigates to a community page which isn't on the user's currently selected network (e.g. when the user is on the Matic network, and they navigate to the "Ethane" community page)? Or maybe that would be too much, and the modal would just annoy the user?